README.md

```diff
@@ -54,7 +54,7 @@ The following GPU-specific containers exist, and can be used only with the speci
 
 | Standard Image   | T4 Optimized Image      |
 |------------------|-------------------------|
-| redact-gpu:2.3.1 | redact-gpu:2.3.1-trt-T4 |
+| redact-gpu:2.4.0 | redact-gpu:2.3.1-trt-T4 |
```

Comment:
I think this should stay as 2.3.1, since this is a comparison between different optimisations of the same image

Reply:
oh okay

Reply:
the commit is dropped